mgf.js

```diff
@@ -1,16 +1,18 @@
-var createHash = require('create-hash');
+var createHash = require('create-hash')
+var Buffer = require('safe-buffer').Buffer
+
 module.exports = function (seed, len) {
-  var t = new Buffer('');
-  var  i = 0, c;
+  var t = Buffer.alloc(0)
+  var i = 0; var c
```

Comment:
this is odd

Reply:
I think it's an artifact of standard maybe, I can fix it